qiskit/extensions/quantum_initializer/initializer.py

```diff
@@ -387,7 +401,9 @@ def initialize(self, params, qubits=None):
         qubits (QuantumRegister or int):
             * QuantumRegister: A list of qubits to be initialized [Default: None].
             * int: Index of qubit to initialized [Default: None].
-
+        normalize (Bool): Function to normalize a params array of real or complex weights.
+                Example list: [1,2,3,4]. Example ndarray: np.array([1+4.j, 3-0.j]).
+            * Bool: Pass True to normalize vector.
```

Comment:
This is another place where Sphinx will error (actually, I think the error I copied it is from this docstring).  See the previous comment, too.